specifyweb/backend/workbench/upload/upload_table.py

```
@@ -60,6 +72,362 @@
+            )
+
+            for rule in _get_uniqueness_rule_configs(
+                model._meta.apps,
+                model.__name__,
+            ):
+                if any(
+                    self._field_matches(changed_names, field_name)
+                    for field_name in rule.all_fields
+                ):
+                    return True
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)